src/query-helpers.js

```diff
@@ -75,18 +86,79 @@ function makeFindQuery(getter) {
     waitFor(() => getter(container, text, options), waitForOptions)
 }
 
+const wrapSingleQueryWithSuggestion = (query, queryAllByName, variant) => (
+  container,
+  ...args
+) => {
+  const element = query(container, ...args)
+  const [{suggest = true} = {}] = args.slice(-1)
+
+  if (getConfig().showSuggestions && suggest) {
```

Comment:
I think that `suggest` should override `showSuggestions`. So maybe it should be:

```suggestion
  const [{suggest = getConfig().showSuggestions} = {}] = args.slice(-1)

  if (suggest) {
```

Reply:
done